BE/src/main/java/kr/codesquad/issuetracker/service/IssueSearchService.java

```diff
@@ -51,4 +71,70 @@ private int countOpened() {
 		return (int) issueRepository.countByStatus(Status.OPEN);
 	}
 
+	@Transactional(readOnly = true)
+	public List<LabelsResponseDto> listLabels() {
```

Comment:
메소드나 변수명에 타입을 나타내는 명칭은 생략해도 좋습니다 :)

```suggestion
	public List<LabelsResponseDto> labels() {
```